scripts/tests/testIntegration.sh

```diff
@@ -4,4 +4,4 @@
 export NODE_ENV='test'
 
 echo 'Start firestore emulator and run integration tests:'
-firebase emulators:exec 'nyc mocha test/integration/**'
+firebase emulators:exec 'nyc mocha test/integration/**' --project='<your firestore app name>'
```

Comment:
No longer needed. Reason -

Reply:
ok!

Reply:
Please remove this as per shubham's comment as he has raised a PR that does the same